### PR TITLE
ACM-2375 Add vendor label

### PR DESCRIPTION
**Pull request by KevinFCormier**
https://issues.redhat.com/browse/ACM-2375

### CODE REVIEW

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/templates/assisted-installer/hypershift-template.hbs [hunk 1]
```diff
@@ -149,6 +150,7 @@ spec:
   clusterNamespace: '{{{hypershift.name}}}'
   clusterLabels:
     cloud: ai-hypershift
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/templates/assisted-installer/hypershift-template.hbs:152]: @rawagner The cloud label is already set to `hybrid` for AI and CIM. But for HyperShift, it is set to `hypershift` on the `ManagedCluster` but `ai-hypershift` here on the `KlusterletAddonConfig`. Should we set this one to `hypershift` as well?

Comment (rawagner) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/templates/assisted-installer/hypershift-template.hbs:152]: I didnt know how these labels are used, so basically these are random values :)
There is just a couple of values recognized by backend https://github.com/stolostron/cluster-lifecycle-api/blob/6f0a02d2aaf571d50008e346bf40cf125200ae15/clusterinfo/v1beta1/types.go#L235-L282

In that regards
`cloud` should be `BareMetal` for all AI/CIM templates IMO.

**Review by RobDolares — APPROVED** [review]
/lgtm

**Pull request merged**